sql/core/src/test/resources/sql-tests/inputs/collations.sql

```diff
@@ -101,3 +101,305 @@ select str_to_map(text collate utf8_binary, pairDelim collate utf8_lcase, keyVal
+select lpad(utf8_binary, 8, utf8_lcase collate utf8_binary) from t1;
+select lpad(utf8_binary collate utf8_lcase, 8, utf8_lcase collate utf8_lcase) from t1;
+select lpad(utf8_binary, 8, 'a'), lpad(utf8_lcase, 8, 'a') from t1;
+select lpad(utf8_binary, 8, 'AaAA' collate utf8_lcase), lpad(utf8_lcase, 8, 'AAa' collate utf8_binary) from t1;
+
+-- Locate
+select locate(utf8_binary, utf8_lcase) from t1;
+select locate(s, utf8_binary) from t1;
+select locate(utf8_binary collate utf8_binary, s collate utf8_lcase) from t1;
+select locate(utf8_binary, utf8_lcase collate utf8_binary) from t1;
+select locate(utf8_binary collate utf8_lcase, utf8_lcase collate utf8_lcase, 3) from t1;
+select locate(utf8_binary, 'a'), locate(utf8_lcase, 'a') from t1;
+select locate(utf8_binary, 'AaAA' collate utf8_lcase, 4), locate(utf8_lcase, 'AAa' collate utf8_binary, 4) from t1;
+
+-- StringTrim*
+select TRIM(utf8_binary, utf8_lcase) from t1;
+select TRIM(s, utf8_binary) from t1;
+select TRIM(utf8_binary collate utf8_binary, s collate utf8_lcase) from t1;
+select TRIM(utf8_binary, utf8_lcase collate utf8_binary) from t1;
+select TRIM(utf8_binary collate utf8_lcase, utf8_lcase collate utf8_lcase) from t1;
+select TRIM('ABc', utf8_binary), TRIM('ABc', utf8_lcase) from t1;
+select TRIM('ABc' collate utf8_lcase, utf8_binary), TRIM('AAa' collate utf8_binary, utf8_lcase) from t1;
+select BTRIM(utf8_binary, utf8_lcase) from t1;
+select BTRIM(s, utf8_binary) from t1;
+select BTRIM(utf8_binary collate utf8_binary, s collate utf8_lcase) from t1;
+select BTRIM(utf8_binary, utf8_lcase collate utf8_binary) from t1;
+select BTRIM(utf8_binary collate utf8_lcase, utf8_lcase collate utf8_lcase) from t1;
+select BTRIM('ABc', utf8_binary), BTRIM('ABc', utf8_lcase) from t1;
+select BTRIM('ABc' collate utf8_lcase, utf8_binary), BTRIM('AAa' collate utf8_binary, utf8_lcase) from t1;
+select LTRIM(utf8_binary, utf8_lcase) from t1;
+select LTRIM(s, utf8_binary) from t1;
+select LTRIM(utf8_binary collate utf8_binary, s collate utf8_lcase) from t1;
+select LTRIM(utf8_binary, utf8_lcase collate utf8_binary) from t1;
+select LTRIM(utf8_binary collate utf8_lcase, utf8_lcase collate utf8_lcase) from t1;
+select LTRIM('ABc', utf8_binary), LTRIM('ABc', utf8_lcase) from t1;
+select LTRIM('ABc' collate utf8_lcase, utf8_binary), LTRIM('AAa' collate utf8_binary, utf8_lcase) from t1;
+select RTRIM(utf8_binary, utf8_lcase) from t1;
+select RTRIM(s, utf8_binary) from t1;
+select RTRIM(utf8_binary collate utf8_binary, s collate utf8_lcase) from t1;
+select RTRIM(utf8_binary, utf8_lcase collate utf8_binary) from t1;
+select RTRIM(utf8_binary collate utf8_lcase, utf8_lcase collate utf8_lcase) from t1;
+select RTRIM('ABc', utf8_binary), RTRIM('ABc', utf8_lcase) from t1;
+select RTRIM('ABc' collate utf8_lcase, utf8_binary), RTRIM('AAa' collate utf8_binary, utf8_lcase) from t1;
+
+drop table t1;
+drop table t2;
+drop table t3;
+drop table t4;
+drop table t5;
+drop table t6;
```

Comment:
can we use other column names, so that we avoid having two "t1" tables in the same file?

Reply:
for example, if we already have t1-t4 in this file, can we just continue from t5